pytorch_lightning/loggers/trains.py

```diff
@@ -327,8 +336,20 @@ def set_bypass_mode(cls, bypass: bool) -> None:
         """
         cls._bypass = bypass
 
+    @classmethod
+    def bypass_mode(cls) -> bool:
+        """
+        bypass_mode returns the bypass mode state.
+        Notice GITHUB_ACTIONS env will automatically set bypass_mode to True
+        unless overridden specifically with set_bypass_mode(False)
+
+        :return: If True, all outside communication is skipped
+        """
+        return cls._bypass if cls._bypass is not None else \
```

Comment:
but this will still fail if a contributor will run it locally without connection, right?

Reply:
Yes working on CI failing + will switch to logger as well :)

Reply:
Is it necessary to doctest the print statements? Perhaps it is enough to just test the instantiation?  Anyway I think having the doctest is already very nice :)

Reply:
@awaelchli the print is to match the expected behavior, so users know what to excepted when everything is working correctly. That said the actual text is printed (stdout) inside Trains package, so my goal was that CI process imitates the Trains prints.
@Borda what do you think? If I'll use logging it will break the doctest again, unless we change the documentation (which I'm okay with) ...

p.s. 
CI finally passing, all in bypass mode

Reply:
oh right, these prints are the package itself. ok sry :)

Reply:
@Borda @williamFalcon again my apologies for not realizing the doctest are executed as part of the CI (after seeing it, I can definitely relate to how annoying tests failing is).
Also, while debugging the CI pipelines I found that the same doctests are executed by three different pipelines: GitHub workflow, Circle, and Drone. Not sure if it is on purpose, but just fyi...

Reply:
> but this will still fail if a contributor will run it locally without connection, right?


If users use the TrainsLogger without a connection the *Trains* package will output an error (it has to have a connection)
That said bypass mode is still supported with:

```python
TrainsLogger.set_bypass_mode(True)
```